pom.xml

```diff
@@ -38,6 +38,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/javax.servlet.jsp.jstl/jstl -->
```

Comment:
Use the java formetere

Reply:
I Used the Standard Formatter @Dalchandsahu